modules/custom/az_paragraphs/src/Plugin/migrate/process/ParagraphsBehaviorSettings.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace Drupal\az_paragraphs\Plugin\migrate\process;
+
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\Plugin\MigratePluginManagerInterface;
+use Drupal\migrate\Plugin\MigrationInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Configure Behavior for paragraphs.
+ *
+ * Examples:
+ *
+ * Consider a paragraph with a behavior plugin.
+ * @code
+ * field_uaqs_setting_text_bg_color_processed:
+ *   - plugin: extract
+ *     source: field_uaqs_setting_text_bg_color
+ *     index:
+ *       - 0
+ *       - value
+ *   - plugin: static_map
+ *     map:
+ *       bg-transparent: transparent
+ *       bg-trans-white: light
+ *       bg-trans-sky: light
+ *       bg-trans-arizona-blue: dark
+ *       bg-trans-black: dark
+ *       dark: dark
+ *       light: light
+ *   - plugin: default_value
+ *     default_value: 'light'
+ * view_mode_processed:
+ *   - plugin: static_map
+ *     source: view_mode
+ *     map:
+ *       uaqs_bg_img_content_left: 'col-md-8 col-lg-6'
+ *       uaqs_bg_img_content_center: 'col-md-8 col-lg-6 col-md-offset-2 col-lg-offset-3'
+ *       uaqs_bg_img_content_right: 'col-md-8 col-lg-6 col-md-offset-4 col-lg-offset-6'
+ *   - plugin: default_value
+ *     default_value: 'col-md-8 col-lg-6'
+ * content_style_processed:
+ *   - plugin: default_value
+ *     default_value: 'column'
+ * bottom_spacing_processed:
+ *   - plugin: default_value
+ *     default_value: 'mb-0'
+ *     source: bottom_spacing
+ * behavior_settings:
+ *   plugin: paragraphs_behavior
+ *   paragraph_behavior_plugins:
+ *     az_text_media_paragraph_behavior:
+ *       bg_color: '@field_uaqs_setting_text_bg_color_processed'
+ *       bg_attach: '@field_uaqs_setting_bg_attach_value'
+ *       position: '@view_mode_processed'
+ *       full_width: '@processed_full_width'
+ *       style: '@content_style_processed'
+ *       az_display_settings:
+ *         bottom_spacing: '@bottom_spacing_processed'
+ *
+ * @endcode * @MigrateProcessPlugin(
+ *   id = "paragraphs_behavior"
```

Comment:
paragraphs_behavior_settings is already taken, so I welcome any feedback on the id of the plugin.

Reply:
Let's add an `az_` prefix to the ID/machine name please.

Reply:
Maybe `az_paragraphs_behavior_settings` (?)